examples/camera-controller/commands/interactive/InteractiveCommands.cpp

```diff
@@ -118,6 +121,7 @@ char * InteractiveStartCommand::GetCommand(char * command)
 
     std::string cmd = sCommandQueue.front();
     sCommandQueue.pop();
+    lock.unlock();
```

Comment:
Do you care that the fabric-admin app presumably has the same bugs that you are fixing here?

Reply:
This is optimization, but it seems not necessary. Drop to align with chip-tool